src/components/Book.js

```diff
@@ -1,19 +1,26 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] **[OPTIONAL]** For the `prop` issue you were having, let me walk you through you can resolve it.

  - Install the `prop-types` by running the command: `npm i prop-types`
  - Remove this eslint-disabling here  
  - In the top of this file, import it like so: `import PropTypes from 'prop-types';`
  - Towards the end of your file, validate your props like so:
    ```
    Book.propTypes = {
      book: PropTypes.shape({
        id: PropTypes.string,
        author: PropTypes.string,
        title: PropTypes.string,
      }).isRequired,
    };
    ```

  You can refer to this React documentation article about [Typechecking with proptypes](https://reactjs.org/docs/typechecking-with-proptypes.html)

Reply:
Thank you very much, man. I appreciate it 👍